msteams-platform/bots/how-to/format-ai-bot-messages.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Bot messages with AI-generated content
+description: Learn how to format your bot with an AI label, sensitivity labels, citations, and feedback buttons built using Teams AI library or Bot Framework SDK.
+ms.topic: conceptual
+ms.localizationpriority: medium
+---
+
+# Bot messages with AI-generated content
+
+AI labels, citations, feedback buttons, and sensitivity labels in your bot’s messages improve user engagement and foster transparency and trust.
+
+* [AI label](#ai-label) enables users to identify that the message was generated using AI.
+* [Citations](#citations) enables users to refer to the source of the bot message through in-text citations and references.
+* [Feedback buttons](#feedback-buttons) enables users to provide positive or negative feedback to the bot messages.
+* [Sensitivity label](#sensitivity-label) enables users to understand the confidentiality of the bot message.
+
+The following screenshots show how bot messages can be enhanced with AI labels, citations, feedback buttons, and sensitivity labels:
+
+# [Before](#tab/before)
+
+:::image type="content" source="../../assets/images/bots/ai-bot-no-features.png" border="false" alt-text="Screenshot shows a bot message with no AI label, citation, feedback buttons, and sensitivity label.":::
+
+# [After](#tab/after)
+
+:::image type="content" source="../../assets/images/bots/ai-bot-message.png" border="false" alt-text="Screenshot shows a bot message with AI label, citation, feedback buttons, and sensitivity label.":::
+
+---
+
+> [!NOTE]
+>
+> * AI label, sensitivity label, citations, and feedback buttons are available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md) for one-on-one and group chats, but aren't supported in channels.
+> * AI label, sensitivity label, citations, and feedback buttons are available only in Teams web and desktop clients.
+> * AI label, sensitivity label, citations, and feedback buttons are available in [Government Community Cloud (GCC), GCC High, and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.
+
+## AI label
+
+An AI label added to your bot message specifies that the message is generated by AI. AI bots use Large Language Models (LLMs) that are generally reliable, however, adding an AI label prevents confusion about the source of the information.
+
+:::image type="content" source="../../assets/images/bots/ai-bot-label.png" border="false" alt-text="Screenshot shows an AI label in a bot message.":::
+
+### Add AI label
+
+For a bot built using **Teams AI library**, an AI label is automatically enabled for all AI-powered bot messages in the `ai` module within the `PredictedSayCommand` action. For more information, see [AIEntity interface](https://github.com/microsoft/teams-ai/blob/main/js/packages/teams-ai/src/actions/SayCommand.ts#L15).
```

Comment:
This url doesn't appear to work: 
<img width="322" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/14900841/c52ff15b-1831-47e7-95b3-2a1fb25c93ca">

Reply:
Tested on multiple devices. The links are working.